### PR TITLE
Update documentation with info on tuning Vertica export perf

### PR DESCRIPTION
**Pull request by alexr-bq**
### Summary

Updated perf-tests documentation to include information on tuning Vertica for the operation. Added link to this from main readme.

### Additional Reviewers

@jonathanl-bq 
@ravjotbrar 
@NerdLogic

### CODE REVIEW

File: performance-tests/README.md [hunk 1]
```diff
@@ -16,12 +16,26 @@ Configuration is specified with application.conf (HOCON format)
...
 
+The biggest factor in connector performance will be resources for Vertica and Spark. Vertica, particularly with default settings may run into a memory bottleneck. This can be improved via configuration of resource pools. 
+
+### Vertica Resource Pool Configuration
+
+The connector's Vertica-to-Spark functionality relies on a query to export data from Vertica to an intermediate filestore. This operation reserves a lot of memory, and the more memory available to it, the more threads it can create to parallelize the operation. 
+
+It is suggested that the resource pool used for the operation is given as much memory as possible, and has its "plannedconcurrency" value set to as low as possible. 
+
+For an explanation of this, any given Vertica in query may only reserve its total provided memory divided by the plannedconcurrency value. A more detailed explanation can be found [here.](https://www.vertica.com/blog/do-you-need-to-put-your-query-on-a-budgetba-p236830/) The plannedconcurrency value sets how many independent queries are expected to be run, and the connector only uses one query at a time. This query is then parallelized by Vertica.
```
Comment (jonathanl-bq) [inline performance-tests/README.md:27]: "any given Vertica in query" is this a typo?

Comment (alexr-bq) [reply performance-tests/README.md:27]: yup

File: performance-tests/README.md [hunk 2]
```diff
@@ -16,12 +16,26 @@ Configuration is specified with application.conf (HOCON format)
...
+
+### Connector Options
+
 There are some connector parameters that may affect the performance of a read from Vertica operation.
 
 num_partitions: Will set how many partitions are created, representing how many parallel executors will be reading data from the intermediate location at once. This should roughly correspond to the processing power / number of cores in the spark cluster.
-max_file_size_export_mb and max_row_group_size_export_mb: Represent configuration of the files exported from Vertica to the intermediary location. These values default to where we find the best export performance lies. However, these can be tweaked dependending on details of the given clusters.
+max_file_size_export_mb and max_row_group_size_export_mb: Represent configuration of the parquet files exported from Vertica to the intermediary location. These values default to where we find the best export performance lies. However, these can be tweaked dependending on details of the given clusters.
```
Comment (jonathanl-bq) [inline performance-tests/README.md:34]: What are the defaults, specifically?

Comment (alexr-bq) [reply performance-tests/README.md]: Added

**Review by jonathanl-bq — APPROVED** [review]

**Pull request merged**